### PR TITLE
[incubator/druid] New Druid Chart

### PR DESCRIPTION
**Pull request by maver1ck**
Signed-off-by: Maciej Bryński <maciej.brynski@xcaliber.com>

#### What this PR does / why we need it:

Apache Druid is popular open source OLAP database and there is no chart for it.

Fixes: https://github.com/apache/incubator-druid/issues/5582

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Variables are documented in the README.md

### CODE REVIEW

**Comment by maver1ck** [comment]
Funny thing.
Error is from mysql tests
```
I1116 12:47:19.773] Error linting and installing charts
I1116 12:47:19.773] ------------------------------------------------------------------------------------------------------------------------
I1116 12:47:19.773]  âď¸ incubator/druid > Pods 'druid-4u225rf2u4-mysql-test' did not reach ready state!
I1116 12:47:19.774] ------------------------------------------------------------------------------------------------------------------------
```
Any idea why ?

**Comment by maver1ck** [comment]
Any chances to get review for this ?

File: incubator/druid/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Apache Druid
+
+[Apache Druid (incubating)](http://druid.io/) is a high performance analytics data store for event-driven data.
+
+
+## Install Chart
+
+To install the Druid Chart into your Kubernetes cluster :
+
+```bash
```
Comment (FrenchBen) [inline incubator/druid/README.md:10]: Is the namespace required here?

File: incubator/druid/README.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+- Druid Overlord
+- Druid Broker
+- Druid Coordinator
+- Druid Historical
+- Druid Middle Manager
+
+
+### Druid configuration
+
+Duid configuration can be changed by using environment variables from Docker image.
```
Comment (FrenchBen) [inline incubator/druid/README.md]: Typo for `Druid`

File: incubator/druid/README.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+See the
+[Druid Docker entry point](https://github.com/maver1ck/druid-docker/blob/historical/start-druid.sh)
```
Comment (FrenchBen) [inline incubator/druid/README.md:53]: Since you mention all configurations below, is this really needed?

**Review by FrenchBen — COMMENTED** [review]
Couple of comments but a good start

**Comment by rahulsingh303** [comment]
I have created a druid helm chart ans tested on both minikube and kubernetes cluster , here's the link https://github.com/rahulsingh303/druid-helm

-Rahul

**Comment by maver1ck** [comment]
@SamClinckspoor 
I'm using this on my production, but I didn't have time recently to make some fixes here.
Do you there is something to be fixed ?

**Comment by SamClinckspoor** [comment]
@maver1ck nothing from my side. would be nice to have this merged as a starting point

**Comment by donbowman** [comment]
Looks great.
the image ... maver1ckpl/druid-docker ... is it the intent to have an 'official' one from apache/druid?

**Comment by maver1ck** [comment]
@donbowman I hope so.

**Comment by donbowman** [comment]
I have PR in process for a docker image https://github.com/apache/incubator-druid/pull/6896

**Pull request merged**